### PR TITLE
Update README.md and include in release

### PR DESCRIPTION
**Pull request by fasmat**
Made language in the README clearer as to what must be done by the operator.

README.md is now included in the release archive.

Additionally added a check that if a user initializes subsets of PoST they must provide an ID.

### CODE REVIEW

**Review by poszu — APPROVED** [review]
Let's wait for @pigmej approval before merging.

**Review by pigmej — APPROVED** [review]
Good additions! :)

**Pull request merged**
